### PR TITLE
Adds a new plant called money tree + nerfs bees instant 2x yield to 1.3x

### PR DESCRIPTION
**Pull request by ma44**
# **If you can actually read the thingy below the changelog before crying out just because ma44 liked some cash lemons**

:cl: ma44
add: Money tree, mutated from lemons
tweak: Bees 2x yield changed to 1.3x yield
/:cl:

[why]: Botany right now has 3 different options
1. Grow food (boring as hell to most people)
2. Grow lots of drugs/meds (mostly just drugs, sometimes doctors want meds but that's an edge case)
3. Grow cool acid tomatos or whatever kind of valid hunting mass producible plant these people are coming up with

With this pr, there will be a 4th option, which is growing some neat cash so you can buy some stuff from cargo to help the crew.

How does this compare to the other three?

1. don't get to fuck over the innocent botanists when shitcurity comes to take all of your stuff as it's *dangerous*
2. Makes you into a bit of jack of all trades, as you can order meds, guns and different items, but using other strategies can make you very efficient at what you want to do (acid tomatos, deathnettles, earthsblood, cloning)
3. Requires more labor than other plants if you want to (attempt) at trying to get more of these, sure you can slap in some gaia, but you will still have to work for mutating them in the first place then trying your luck at improving stats. 

## **Important balance thingies that will totally blow your mind on how this is not even overpowered**

It's impossible to get seeds from this, as it doesn't drop a food product or anything, just cash, meaning you CAN'T SLAP IT INTO A GENE MODIFIER OR CAN'T GET SEEDS
This is really important, as with cash lemons, you could just simply min max and get all the seeds you want, now with this, you need to pour in mutagen (only a 1/2 chance of getting it anyway) then pouring more stuff until you get what you want. This also provides MUCH less money (200 credits at 1 HUNDRED potency)

Also floral gun yield projectile won't effect this, as then it would be easier to get yield 10 with virtually no risks (mutation is essentially just mutagen, can still downgrade your plant)

For the bees, it's ridiculous that they can simply twice the yield of a plant just by visiting it once, so it's being brought down to 1.3 (and also for this pr somewhat)

(end of long ass rant)

### CODE REVIEW

File: code/modules/hydroponics/seeds.dm [hunk 1]
```diff
@@ -142,6 +143,21 @@
 	var/list/result = list()
 	var/output_loc = parent.Adjacent(user) ? user.loc : parent.loc //needed for TK
 	var/product_name
+	if(plantname = "Money Tree")
```
Comment (ChangelingRain) [inline code/modules/hydroponics/seeds.dm]: No.

**Comment by ChangelingRain** [comment]
Cargo already has a mechanic to give cargo points, by giving seeds of rare plants to cargo. They don't need another one, especially one that's as unusable as this is.

**Comment by ma44** [comment]
damm

**Comment by FantasticFwoosh** [comment]
😦

**Pull request closed without merging**